### PR TITLE
Venue Page

### PR DESCRIPTION
**Pull request by jimzarkadas**
This is a first version of Venue Page (#34). Feel free to tell me your opinion and suggest any improvements!

### CODE REVIEW

File: source/stylesheets/_venue.css.scss [hunk 1]
```diff
@@ -0,0 +1,113 @@
+#venue-page {
+  width: 100%;
+  padding-bottom: 20px;
+  background-color: #f2f2f2;
+  font-family: "Ubuntu",Arial,"Helvetica Neue",Helvetica,Geneva,sans-serif;
```
Comment (fotos) [inline source/stylesheets/_venue.css.scss:5]: It would be much better to use already defined SASS variables. :wink:

Other than that well done.

Comment (jimzarkadas) [inline source/stylesheets/_venue.css.scss:5]: Should I make a new pull request with additions ?

Comment (fotos) [inline source/stylesheets/_venue.css.scss:5]: Feel free to edit the site directly. You now have push access.

Open source etiquette applies. Don't make silly things, discuss broad changes first, feel free to fix mistakes and errors.

Comment (avgerin0s) [inline source/stylesheets/_venue.css.scss:5]: :+1: @fotos

Comment (fotos) [inline source/stylesheets/_venue.css.scss:5]: Oh I forgot to mention that @eavgerinos, you too, have push access as well! Enjoy.

**Comment by fotos** [comment]
@jimzar @eavgerinos the [venue page is online](http://euruko2013.org/venue) and linked from the header, the footer and the [venue section in the first page](http://euruko2013.org/#venue).

I fixed a couple of things, if you want to [check them out](/euruko2013/site/commit/e8bdafd6657fa8ef7865622bd1f38ac4255fd1ed).

Thanks for all the work you put into it. :smiley: :+1:

**Comment by avgerin0s** [comment]
@fotos Nice. Styles are much more DRY. Text is more readable. :+1:

**Comment by jimzarkadas** [comment]
Nice job! (& footer ) :P

**Pull request merged**